data_steward/tools/add_hpo.py

```diff
@@ -1,19 +1,28 @@
 """
-Add a new HPO site to config file and BigQuery lookup tables
+Add a new HPO site to config file and BigQuery lookup tables and adds the new masked src_id and hpo_id to
+    the pipeline_tables.site_maskings table
 
 Note: GAE environment must still be set manually
 """
+# Python imports
 import logging
 import csv
 
+# Third party imports
 from googleapiclient.errors import HttpError
 import pandas as pd
 
+# Project imports
 import bq_utils
 import resources
+import gcs_utils
+import app_identity
 import constants.bq_utils as bq_consts
+from utils import bq
+from cdr_cleaner.cleaning_rules.deid.generate_site_mappings_and_ext_tables import GenerateSiteMappingsAndExtTables, \
+    SITE_TABLE_ID
 from tools import cli_util
-import gcs_utils
+from common import JINJA_ENV, PIPELINE_TABLES
 
```

Comment:
Maybe we should add pipeline_logging like you did in `generate_site_mappings_and_ext_tables.py`?